### ISSUE TITLE
Start generating XLIFF data from System.Xaml

### ISSUE THREAD

**Issue by rladuca** [dotnet/wpf#171]
We are not currently setup to generate XLIFF data.

We need to configure the repo to do so as a first step in the localization pipeline.

**Comment by rladuca** [comment]
Re-opening as we got further guidance from Arcade that we should be marking as IsShipping rather than working around the conditional in localization.targets.
